.swagger-codegen/VERSION

```diff
@@ -1 +1 @@
-2.4.19
\ No newline at end of file
+2.4.27
```

Comment:
The script from the [README](https://github.com/TeePublic/teepublic-search#update-service-interface) helps us keep the swagger version from changing

Reply:
Which script? These steps?
https://github.com/TeePublic/teepublic-search/blob/master/README.md?plain=1#L126-L130
I am happy to download that specific version rather than using the latest 2.4.x version, but if we are ok with upgrading the version, then these updates are deliberate.

Reply:
If you'd like to take on updating the swagger version as well, that'll include an update to the README and the .jar file as well as any routine version upgrade monitoring post-rollout

Reply:
Ah, didn't expect us to have compiled artifacts merged in.
I'll rerun these with that for now and create another PR for upgrading sort of stuff